mobile-app/app/components/BottomSheetTokenList.tsx

```diff
@@ -27,7 +27,7 @@ export interface BottomSheetToken {
   id: string
```

Comment:
Irrelevant to this PR, but probably worth noting, there is circular dep here L13. (component <-> screen)
Naming wise, this bottomsheet seems to be collateral token specific instead of general token.